README.md

```diff
@@ -443,6 +464,8 @@ std-toast {
 }
 ```
 
+#### Action button
```

Comment:
```suggestion
#### Action
```

The rest of our language tries to be agnostic about button, anchor, or some other custom element slotted to action.